ui/react-components/components/Dropdown/AsyncDropdown.test.jsx

```diff
@@ -0,0 +1,214 @@
+import {fireEvent, waitForElement} from "@testing-library/react";
+import React from "react";
+import AsyncDropdown from "./AsyncDropdown.jsx";
+import selectEvent from "react-select-event";
+import '@testing-library/jest-dom/extend-expect';
+import {renderWithReactIntl} from '../../utils/TestUtil';
+
+const loadOptions = (inputValue, callback) => {
+    // eslint-disable-next-line angular/timeout-service
+    setTimeout(() => {
+        callback(filterColors(inputValue));
+    }, 1000);
+};
+
+const filterColors = (inputValue) => {
```

Comment:
Prefer using domain name rather than generic names like `inputValue`

Reply:
Will change the name